storage/innobase/include/buf0buf.h

```diff
@@ -1301,6 +1298,11 @@ class buf_pool_t
   /** Resize from srv_buf_pool_old_size to srv_buf_pool_size. */
   inline void resize();
 
+  /** Collect garbage (release pages from the LRU list) */
+#ifdef __linux__
+  inline void garbage_collect();
```

Comment:
I think that Doxygen could complain about the stray comment if not run with `-D__linux__`. Better move it inside the `#ifdef`.